helmfile/upstream/cert-manager/values.yaml

```diff
@@ -39,7 +39,7 @@ global:
     # renewal of a leadership.
     # retryPeriod: 15s
 
-installCRDs: false
+installCRDs: true
```

Comment:
This doesn't seem to match the [upstream value](https://github.com/jetstack/cert-manager/blob/v1.6.1/deploy/charts/cert-manager/values.yaml#L42)

Reply:
Fixed!